### PR TITLE
Fix ESP32 build

### PR DESCRIPTION
**Pull request by vikramdattu**
1. Fixed ESP32 build.
Build was broken after a commit which makes makefile inc file to be named as `makefile_$(TARGET).inc` (see https://github.com/tensorflow/tensorflow/issues/43898 for more details)

2. Modified README files for ESP32 to make those consistent across other examples.

### CODE REVIEW

**Comment by vikramdattu** [comment]
@petewarden PTAL.

cc: @AdityaHPatwardhan @kedars

File: tensorflow/lite/micro/tools/make/targets/esp32_makefile.inc [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# Settings for Espressif ESP32
-
-ifeq ($(TARGET), esp)
```
Comment (vikramdattu) [inline tensorflow/lite/micro/tools/make/targets/esp32_makefile.inc:3]: No need for this check! As the file is picked only if it is esp build.

Removed this in renamed makefile `esp_makefile.inc`.

File: tensorflow/lite/micro/examples/micro_speech/README.md [hunk 2]
```diff
@@ -214,28 +214,36 @@ The next steps assume that the
 
 ### Generate the examples
 
-The example project can be generated with the following command: `make -f
-tensorflow/lite/micro/tools/make/Makefile TARGET=esp
-generate_micro_speech_esp_project`
+The example project can be generated with the following command:
+```
+make -f tensorflow/lite/micro/tools/make/Makefile TARGET=esp generate_micro_speech_esp_project
+```
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/micro_speech/README.md:220]: this formatting will likely get un-done by the internal CI system.

Adding this line to the top of the file should help:
```md
<!-- mdformat off(b/169948621#comment2) -->
```

similar to https://github.com/tensorflow/tensorflow/commit/c702a443f55171b8d3bddc4a02980d6ad3fc9fea

File: tensorflow/lite/micro/examples/person_detection/esp/README_ESP.md [hunk 3]
```diff
@@ -40,15 +40,22 @@ idf.py build
 
 ### Load and run the example
 
-To flash (replace `/dev/ttyUSB0` with the device serial port): `idf.py --port
-/dev/ttyUSB0 flash`
+To flash (replace `/dev/ttyUSB0` with the device serial port):
+```
+idf.py --port /dev/ttyUSB0 flash
+```
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/person_detection/esp/README_ESP.md:46]: same comment.

Adding this line to the top of the file should help:
```md
<!-- mdformat off(b/169948621#comment2) -->
```

File: tensorflow/lite/micro/examples/person_detection/README.md [hunk 4]
```diff
@@ -302,33 +302,45 @@ The next steps assume that the
 
 ### Generate the examples
 
-The example project can be generated with the following command: `make -f
-tensorflow/lite/micro/tools/make/Makefile TARGET=esp
-generate_person_detection_esp_project`
+The example project can be generated with the following command:
+```
+make -f tensorflow/lite/micro/tools/make/Makefile TARGET=esp generate_person_detection_esp_project
+```
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/person_detection/README.md:308]: same comment.

Adding this line to the top of the file should help:
```md
<!-- mdformat off(b/169948621#comment2) -->
```

**Review by advaitjain — CHANGES_REQUESTED** [review]

File: tensorflow/lite/micro/examples/micro_speech/README.md (hunk 2 above)
Comment (vikramdattu) [reply tensorflow/lite/micro/examples/micro_speech/README.md:222]: Thanks, that explains line breaks in READMEs!

**Review by advaitjain — APPROVED** [review]

**Comment by gbaned** [comment]
@vikramdattu  Can you please resolve conflicts? Thanks!

**Comment by vikramdattu** [comment]
> @vikramdattu Can you please resolve conflicts? Thanks!

Done! 👍🏻

**Review by advaitjain — APPROVED** [review]
For the current CL, this is not an issue, but in the future please on merge and [avoid rebasing and force pushes](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/CONTRIBUTING.md#during-the-pr-review) during the PR review.

It is ok to have merge commits -- the github review tool handles those gracefully.

**Review by gbaned — APPROVED** [review]

**Pull request merged**